### PR TITLE
Explorer refactor

### PR DESCRIPTION
**Pull request by octavioamu**
##### Description

Refactor explorer to user vuejs and simplify the work around the js logic.

### CODE REVIEW

File: app/assets/v2/js/explorer.js [hunk 1]
```diff
@@ -0,0 +1,276 @@
+let bounties = [];
+let featuredBounties = [];
+let bountiesPage = 1;
+let BountiesLimit = 10;
+let BountiesOffset = 0;
+let bountiesNumPages = '';
+let bountiesHasNext = false;
+let numBounties = '';
+// let funderBounties = [];
```
Comment (danlipert) [inline app/assets/v2/js/explorer.js]: leftover comment

File: app/assets/v2/js/explorer.js [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+        }
+        vm.numBounties = response.length;
+        if (vm.numBounties < vm.BountiesLimit) {
+          vm.bountiesHasNext = false;
+        } else {
+          vm.bountiesHasNext = true;
+        }
+        if (vm.bountiesHasNext) {
+          vm.BountiesOffset = vm.BountiesOffset + vm.BountiesLimit;
+
```
Comment (danlipert) [inline app/assets/v2/js/explorer.js]: The style here in regards to spacing could be cleaned up - unless this has some implied meaning in Vue I'm unaware of

File: app/assets/v2/js/explorer.js [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+          vm.params[p[0]] = p[1];
+        }
+      }
+    },
+    searchBounties: function() {
+      let vm = this;
+
+      vm.bounties = [];
+      vm.featuredBounties = [];
+      vm.fetchBounties(0);
```
Comment (danlipert) [inline app/assets/v2/js/explorer.js]: Is there any way we can use named arguments for these type of function calls? I feel like its hard to figure out what this is happening without looking up the function definition. Just a general question though... In the past I've done things like name the function `fetchBountiesStartingWithPageAndIsFeatured` although thats a bit extreme

File: app/retail/templates/shared/footer_scripts.html [hunk 4]
```diff
@@ -23,9 +23,9 @@
 </script>
 <script src="{% url 'javascript-catalog' %}"></script>
 <script src="{% static "v2/js/lib/jquery.js" %}"></script>
-{% if user.is_authenticated and env == 'prod' %}
+{% if env == 'prod' %}
   <script src="https://cdn.jsdelivr.net/npm/vue"></script>
-{% elif user.is_authenticated %}
+{% else %}
   <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
```
Comment (danlipert) [inline app/retail/templates/shared/footer_scripts.html]: random question - should we pin this version or bring it into our repo as we do with other libs?

**Comment by thelostone-mc** [comment]
Closing this out for now ! Will keep branch alive if decide to revisit this at a later point in time

**Pull request closed without merging**